README.md

```diff
@@ -12,13 +12,14 @@ into the IntelliJ IDEA UI. Currently this includes:
 * The [Google Cloud Debugger](https://cloud.google.com/tools/cloud-debugger/) 
   The Cloud Debugger can inspect the state of a Java application running on 
   [Google App Engine](https://cloud.google.com/appengine/)
-  at any code location without stopping the application. 
+  at any code location without stopping the application.
+* [Google App Engine](https://cloud.google.com/appengine/docs/) deployment via the Cloud SDK.
```

Comment:
Nah but I will send another one.

On Wed, Aug 10, 2016 at 2:21 PM, Etan Shaul <notifications@github.com>
wrote:

> In README.md
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/816#discussion_r74301039>
> :
>
> > @@ -12,13 +12,14 @@ into the IntelliJ IDEA UI. Currently this includes:
> >  * The [Google Cloud Debugger](https://cloud.google.com/tools/cloud-debugger/)
> >    The Cloud Debugger can inspect the state of a Java application running on
> >    [Google App Engine](https://cloud.google.com/appengine/)
> > -  at any code location without stopping the application.
> > +  at any code location without stopping the application.
> > +* [Google App Engine](https://cloud.google.com/appengine/docs/) deployment via the Cloud SDK.
>
> ok lgtm. Did you want to add that link in PR?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/816/files/33560ec23419489e74e2beb7e8ac37b49b08b295#r74301039>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHf5He-gakCYflfv1HZz9RfciWrvkVBvks5qehazgaJpZM4JhXcc>
> .
>